<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTelaPrincipal</class>
 <widget class="QMainWindow" name="frmTelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataMusik v0.1 - Tela Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frmTelaPrincipal
{
	border-image: url(:/Imagens/Backgrounds/DataMusikLogo.png) 0 0 0 0 stretch stretch;
}


#lblUsuario
{
	font: 20pt &quot;URW Gothic L&quot;;
}

#lblData
{
	font: 16pt &quot;URW Gothic L&quot;;
}

QMenuBar
{
	font: bold 14pt &quot;URW Gothic L&quot;;
	
	background-color: rgb(0, 0, 0, 50);
}

QMenu
{
	font: 45 12pt &quot;URW Gothic L&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
	font: 45 14pt &quot;URW Gothic L&quot;;
}</string>
   </property>
   <widget class="QLabel" name="lblUsuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>51</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Bem vindo(a), </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblData">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Data : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mbrBarraMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <addaction name="actCadastroAcessorio"/>
    <addaction name="actCadastroInstrumento"/>
    <addaction name="actCadastroMidia"/>
    <addaction name="actCadastroUsuario"/>
   </widget>
   <widget class="QMenu" name="menConsuta">
    <property name="title">
     <string>Consulta</string>
    </property>
    <addaction name="actConsultaAcessorio"/>
    <addaction name="actConsultaInstrumento"/>
    <addaction name="actConsultaMidia"/>
    <addaction name="actConsultaSessao"/>
    <addaction name="actConsultaUsuario"/>
   </widget>
   <widget class="QMenu" name="menOutros">
    <property name="title">
     <string>Outros</string>
    </property>
    <addaction name="actManual"/>
    <addaction name="actSobre"/>
    <addaction name="actSessaoAtual"/>
    <addaction name="separator"/>
    <addaction name="actSair"/>
   </widget>
   <addaction name="menCadastro"/>
   <addaction name="menConsuta"/>
   <addaction name="menOutros"/>
  </widget>
  <widget class="QToolBar" name="tbrBarraFerramenta">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actCadastroAcessorio">
   <property name="text">
    <string>Acessório</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Acessórios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actCadastroInstrumento">
   <property name="text">
    <string>Instrumento</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Instrumentos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actCadastroMidia">
   <property name="text">
    <string>Midia</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Mídias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actCadastroUsuario">
   <property name="text">
    <string>Usuário</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Usuários</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actConsultaAcessorio">
   <property name="text">
    <string>Acessório</string>
   </property>
   <property name="toolTip">
    <string>Consulta de Acessórios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actConsultaInstrumento">
   <property name="text">
    <string>Instrumento</string>
   </property>
   <property name="toolTip">
    <string>Consulta de Instrumentos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actConsultaMidia">
   <property name="text">
    <string>Mídia</string>
   </property>
   <property name="toolTip">
    <string>Consulta de Mídias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actConsultaSessao">
   <property name="text">
    <string>Sessão</string>
   </property>
   <property name="toolTip">
    <string>Consulta de Sessões</string>
   </property>
  </action>
  <action name="actConsultaUsuario">
   <property name="text">
    <string>Usuário</string>
   </property>
   <property name="toolTip">
    <string>Consulta de Usuários</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="actManual">
   <property name="text">
    <string>Manual do Usuário</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+M</string>
   </property>
  </action>
  <action name="actSessaoAtual">
   <property name="text">
    <string>Sessão Atual</string>
   </property>
   <property name="toolTip">
    <string>Dados da Sessão Atual </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
  <action name="actSobre">
   <property name="text">
    <string>Sobre...</string>
   </property>
   <property name="toolTip">
    <string>Sobre o DataMusik</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actSair">
   <property name="text">
    <string>Sair...</string>
   </property>
   <property name="toolTip">
    <string>Deslogar do Sistema</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
